homework/index.js

```diff
@@ -30,13 +33,94 @@
     return elem;
   }
 
+  // deletes the previous repository table if selected repository changes
+  function removeChildElements(parent) {
+    while (parent.firstChild) {
+      parent.removeChild(parent.firstChild);
+    }
+  }
+  // to show the information of selected repository in a table
+  function showRepoInfo(index, repoDiv, repoArray, containerDiv) {
+    removeChildElements(repoDiv);
+    const selectedRepo = repoArray[index];
+    const headerArray = ['Repository', 'Description', 'Forks', 'Updated'];
+    const repoInfo = [
+      selectedRepo.name,
+      selectedRepo.description,
+      selectedRepo.forks,
+      selectedRepo.updated_at,
+    ];
+    const table = createAndAppend('table', repoDiv);
+    for (let i = 0; i < headerArray.length; i++) {
+      const tr = createAndAppend('tr', table);
+      createAndAppend('td', tr, { text: headerArray[i], class: 'label' });
+      if (i === 0) {
+        const repoLink = createAndAppend('td', tr);
+        createAndAppend('a', repoLink, {
+          href: `${selectedRepo.svn_url}`,
```

Comment:
No need to quote this since its just a variable, `href: selectedRepo.svn_url,`

Reply:
I checked and fixed similar faults.